debian/postinst

```diff
@@ -2,10 +2,10 @@
 set -e
 
 case "$1" in
+  configure)
   # Unmask the service if it was previously masked.
   # rshim service will be masked after "dpkg -i" or "apt remove" before 2.0.30
   deb-systemd-helper unmask 'rshim.service' >/dev/null || true
```

Comment:
Can we add two more spaces for these three lines?

Reply:
Good catch. Pushed.